std_msgs/Header header
string cuda_version
string driver_version
uint16 gpu_count
rcp/NvidiaGpuDevice[] gpus